x/permission/keeper/keeper.go

```diff
@@ -570,3 +598,27 @@ func (k Keeper) RemoveExpiredPolicies(ctx sdk.Context) {
 		}
 	}
 }
+
+func (k Keeper) MigrateAccountPolicyForResources(ctx sdk.Context) {
+	store := ctx.KVStore(k.storeKey)
+	// PolicyForAccountKey:
+	// ResourcePolicyForAccountPrefix| ResourceID | AccountAddr
+	// iterator key:
+	// ResourceID | AccountAddr
+	migration := func(resourcePolicyForAccountPrefix []byte, resourceType resource.ResourceType) {
+		resourceAccountPolicyStore := prefix.NewStore(store, resourcePolicyForAccountPrefix)
+		iterator := resourceAccountPolicyStore.Iterator(nil, nil)
+		for ; iterator.Valid(); iterator.Next() {
+			resourceIDBz := iterator.Key()[:len(iterator.Key())-sdk.EthAddressLength]
```

Comment:
This will include the  resource type in `resourceIDBz`

Reply:
No, the type is not in the iterator key.